vendor/github.com/goji/httpauth/.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: go
+sudo: false
+
+matrix:
+  include:
+    - go: 1.2
+    - go: 1.3
+    - go: 1.4
+    - go: 1.5
```

Comment:
This `.travis.yml` only has effect on package `httpauth`, which does not use vendor. Test for main repository does not look into these `travis.yml`s.